draft-ietf-taps-interface.md

```diff
@@ -1659,6 +1659,8 @@ a Connection must provide reliability to allow setting an infinite value for the
 lifetime property of a Message. Sending a Message with Message Properties
 inconsistent with the Selection Properties of the Connection yields an error.
 
+Connection Properties describe the default behavior for all Messages on a Connection. If a Message Property contradicts a Connection Property, and if this per-Message behavior can be supported, it overrides the Connection Property for the specific Message. For example, if `Reliable Data Transfer (Connection)` is set to `Avoid` and a protocol with configurable per-Message reliability is used, setting `Reliable Data Transfer (Message)` to `Prefer` for a particular Message will ensure that this Message is reliably delivered. If the underlying protocol can not support this per-Message behavior, this Message Property choice will yield an error.
+
```

Comment:
Reliable Data Transfer (Message) is a Boolean so the 'Prefer' does not make sense. Also in the description of the property is says 

>  Changing the Reliable Data Transfer property on Messages is only possible for Connections that were established with the Selection Property Configure Per-Message Reliability enabled. When this is not the case, changing it will generate an error.

So maybe the text about the error should also be aligned.

Reply:
Good catch, thanks!  Fixing the "type should be Boolean" problem, and regarding the error, I resorted to repeating the statement that we already have at the end of this paragraph ("Note that...").

Reply:
Thanks! Reading it again I am wondering if is is not better to swap the example though? Setting the connection property to required and the per message property to false? I think it will read better.

Reply:
Done - better now?

Reply:
Suggested change: cause --> allow
Also I think the part about the error can actually be removed. I think the error should rather come if the Per-Message Reliability property is not set to enabled. At least this is what the description later in the text says.

Reply:
At least for me, the last sentence would need to be more like: 
"If the underlying protocol can not support a required per-Message behavior, this Message Property choice will yield an error." - since prefer would not yield an error.

Reply:
@abrunstrom done; I did remove the part about the error as you suggested, so I think that also addresses @gorryfair's suggestion in some sense (he suggested to rephrase the sentence, but I removed it now  :)  )